core/scripts/cli/sponsor/create.js

```diff
@@ -6,27 +7,32 @@ const create = async (web3, artifacts, emp) => {
   const ExpandedERC20 = artifacts.require("ExpandedERC20");
   const { toWei, fromWei, toBN } = web3.utils;
```

Comment:
Do you still need the `toBN` import now? If there are any `toBN`'s remaining, maybe just replace them with BigNumber

Reply:
Removed.